New package: GeoSpatialRelations v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: GeoSpatialRelations
- Repository: https://github.com/adribrune/GeoSpatialRelations.jl
- Created by: @adribrune
- Version: v0.1.0
- Commit: cefcf370bbf174fa5233ad240a069511788d5c78
- Reviewed by: @adribrune
- Reference: https://github.com/adribrune/GeoSpatialRelations.jl/issues/5#issuecomment-2519624079
<!-- bf0c69308befbd3ccf2cc956ac8a4671113b158fd87a6d4e741b0173dab051a48f7dc43e593ffff4357bc647a3a835982a4affe98ee80ad288bdfc08260018009f1ca6d1ba5f797f8dfaa4d01bde636edeb3bedee253a44f005e040fceb0a0b807dbae809f8a6eac207d1a857b9ee420d83e53acfd4416a38871bc939063f42ddc63bd1f65495867e94a18fe728ff2b7c8a6e6309835a881c9e9c971d32da38176937326527fd6a4ec84b91c950dfa49 -->

adribrune commented:
stop

goerz commented:
[noblock] If you want to come back to this later: Please make sure that the documentation at https://adribrune.github.io/GeoSpatialRelations.jl/dev/ is linked somewhere from the README.

adribrune commented:
@JuliaRegistrator register

JuliaRegistrator committed:
New package: GeoSpatialRelations v0.1.0

UUID: 4f4b3779-db02-45a2-a250-b7461b60f8cf
Repo: https://github.com/adribrune/GeoSpatialRelations.jl.git
Tree: e8867bfc5673a027224b070e8669aabd97d1fa60

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
